src/main/java/uk/ac/ebi/eva/submission/controller/submissionws/SubmissionController.java

```diff
@@ -87,6 +88,7 @@ public ResponseEntity<?> markSubmissionUploaded(@RequestHeader("Authorization")
         }
 
         Submission submission = this.submissionService.markSubmissionUploaded(submissionId);
+        submissionService.sendMailNotificationForStatusUpdate(submissionAccount, submissionId, SubmissionStatus.UPLOADED, Boolean.TRUE);
```

Comment:
Just curious, is there a reason to use a `Boolean` object here instead of a primitive?

Reply:
Not really and frankly not needed. Have changed to primitive to be consistent